metrics-operator/controllers/metrics/controller.go

```diff
@@ -142,3 +170,111 @@ func (r *KeptnMetricReconciler) fetchProvider(ctx context.Context, namespacedMet
 	}
 	return provider, nil
 }
+
+func aggregateValues(stringSlice []string, aggFunc string) (string, error) {
+	floatSlice, err := stringSliceToFloatSlice(stringSlice)
+	if err != nil {
+		return "", err
+	}
+	var aggValue string
+	switch aggFunc {
+	case "max":
+		aggValue = fmt.Sprintf("%v", calculateMax(floatSlice))
+	case "min":
+		aggValue = fmt.Sprintf("%v", calculateMin(floatSlice))
```

Comment:
when moving to Go 1.21 we can use the slices package :) 
https://pkg.go.dev/slices@master

Reply:
For the aggregation functions?

Reply:
yes :)